main.go

```diff
@@ -189,7 +190,30 @@ func main() {
 		}
 	}()
 
-	purchaser, err := newTicketPurchaser(cfg, dcrdClient, dcrwClient)
+	purchaser, err := ticketbuyer.NewTicketPurchaser(&ticketbuyer.Config{
```

Comment:
i'd rather you make a ticketCfg := ticketbuyer.Config and pass that in as an arg